docs/user/guides/build/webhooks.rst

```diff
@@ -1,37 +1,18 @@
-How to setup build notifications and webhooks
-=============================================
+How to setup webhook notifications
+==================================
 
-In this guide, you can learn how to setup a number of build notification mechanisms.
+In this guide, you can learn how to setup build notification with webhooks.
 Build notifications can alert you when your builds fail so you can take immediate action.
 
 .. note::
 
-   Currently we don't send notifications or trigger :term:`webhooks <webhook>`
-   on :doc:`builds from pull requests </pull-requests>`.
+   Currently we don't trigger :term:`webhooks <webhook>` on :doc:`builds from pull requests </pull-requests>`.
 
 
 .. tip::
     :doc:`/pull-requests`
-        Similarly to build notifications, you can also configure automated feedback for your pull requests.
-
-
-.. contents:: Contents
-    :local:
-
-
-Email notifications
--------------------
-
-Read the Docs allows you to configure emails that can be sent on failing builds.
-This makes sure you know when your builds have failed.
-
-Take these steps to enable build notifications using email:
-
-* Go to :guilabel:`Admin` > :guilabel:`Notifications` in your project.
-* Fill in the **Email** field under the **New Email Notifications** heading
-* Submit the form
-
-You should now get notified by email when your builds fail!
+        Similarly to webhook notifications,
```

Comment:
We should remove this, we already mention PR builds in the above line.

Reply:
I changed this a bit. I moved the note further down where the mechanisms of the trigger are mentioned. That way, the note about _not triggering on pull requests_ has a way better context.